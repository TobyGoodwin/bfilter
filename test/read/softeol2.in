Dull: header

one soft eol line =
and another at end=
