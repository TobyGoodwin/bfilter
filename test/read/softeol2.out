{one soft eol line and another at end}
